libs/vkd3d/device.c

```diff
@@ -2684,7 +2713,11 @@ static HRESULT d3d12_device_create_vkd3d_queues(struct d3d12_device *device,
 }
 
 #define VKD3D_MAX_QUEUE_COUNT_PER_FAMILY (4u)
-static float queue_priorities[] = {1.0f, 1.0f, 1.0f, 1.0f};
+
+/* The queue priorities list contains VKD3D_MAX_QUEUE_COUNT_PER_FAMILY + 1 priorities
+ * because it is possible for low latency to add an additional queue for out of band work
```

Comment:
This doesn't seem to be the case if you consider line 2665 where it removes a queue. I'd rather add one more queue here than removing a queue, since removing a queue may cause unwanted serialization for games that already happen to use all 4 queues for whatever reason.

Reply:
Or maybe I'm just misreading the code here ...